app/lib/features/chat_ng/widgets/reactions/reaction_chips_widget.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:acter/common/extensions/acter_build_context.dart';
+import 'package:acter/common/themes/app_theme.dart';
+import 'package:acter/features/chat_ng/providers/chat_room_messages_provider.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart'
+    show ReactionRecord;
+import 'package:flutter/material.dart';
+
+class ReactionChipsWidget extends StatelessWidget {
```

Comment:
This part is logically same and pasted from legacy but more linear and structured.